### ISSUE TITLE
`COOLEST` support in `lenstronomy`

### ISSUE THREAD

**Issue by ajshajib** [lenstronomy/lenstronomy#452]
### Discussed in https://github.com/lenstronomy/lenstronomy/discussions/451

<div type='discussions-op-text'>

<sup>Originally posted by **ajshajib** April 21, 2023</sup>
It would be convenient to have a feature that packs (and loads) the common dictionaries, lists, `kwargs_data`, `kwargs_psf`, etc. created/used during modeling into a file in the [`COOLEST`](https://github.com/aymgal/COOLEST) format. @LyneVdV informed me that she already has some code that performs this task. 🙏 We can discuss here to figure out where to add such code within `lenstronomy` (e.g., `Util`?), and what would be the requirements/requests for this feature.</div>

**Comment by ajshajib** [comment]
@LyneVdV I have created this issue to track your potential PR to add `COOLEST` support in `lenstronomy`. Please send the PR when it's ready.

**Comment by LyneVdV** [comment]
Great thank you ! Implementing my functions in lenstronomy was not too complicated but I still need to write several test functions. I am working on it on a branch on my git hub :) !

**Comment by LyneVdV** [comment]
I will let you have a look at the PR. Basically, the interface (_coolest_interface.py_) in itself is 2 functions: 

1. one which reads a COOLEST json file and create the necessary lenstronomy kwargs, `create_lenstro_from_coolest`
2. one which updates a COOLEST json file with lenstronomy results, `update_coolest_from_lenstro`

The second one has 2 main purposes: it can save a `kwargs_result`, but it can also save percentile, mean , and median based on an MCMC chain. The later requires the _chain_list_ to be written in a more "user-friendly way" with all the linear, fixed, joint parameters explicitly written. Hence I created another function `create_kwargs_mcmc_from_chain_list` which is just lenstronomy functions put together.
Note that the interface needed side functions for each mass and light model, for reading from and updating to the COOLEST conventions, hence you will see 2 other files _coolest_read_util.py_ and _coolest_update_util.py_ .

In terms of tests, let me know if this is sufficient or not ! I had to add a COOLEST template to be able to test the functions. To avoid storing fake images in the test folder, I created mock lenstronomy images when needed.

To be improved/added:
- have more mass and light profile supported by COOLEST and the interface
- have an example notebook in the "lenstronomy tutorials" (a bit like the `test_full`)
- have functions to create COOLEST json templates (maybe @aymgal already has functions like this)
- TBD
